cron/worker/main.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Vershandlern 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permisshandlerns and
+// limitathandlerns under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"sync"
+
+	"github.com/google/go-github/v32/github"
+	"github.com/ossf/scorecard/checks"
+	"github.com/ossf/scorecard/cron/config"
+	"github.com/ossf/scorecard/cron/data"
+	"github.com/ossf/scorecard/cron/pubsub"
+	"github.com/ossf/scorecard/pkg"
+	"github.com/ossf/scorecard/repos"
+	"github.com/ossf/scorecard/roundtripper"
+	"github.com/shurcooL/githubv4"
+	"go.uber.org/zap"
+)
+
+func processRequest(ctx context.Context,
+	batchRequest *data.ScorecardBatchRequest, bucketURL string,
+	httpClient *http.Client, githubClient *github.Client, graphClient *githubv4.Client) error {
+	repoURLs := make([]repos.RepoURL, len(batchRequest.GetRepos()))
+	var resultCh chan repos.RepoResult
+	var buffer bytes.Buffer
+	checksToRun := checks.AllChecks
```

Comment:
We would have to ignore checks for branch-protection.

https://github.com/ossf/scorecard/blob/main/cron/main.go#L100

Reply:
Done, thanks.